SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -103,7 +104,8 @@ CIncEulerSolver::CIncEulerSolver(CGeometry *geometry, CConfig *config, unsigned
 
   nDim = geometry->GetnDim();
 
-  nVar = nDim+2; nPrimVar = nDim+9; nPrimVarGrad = nDim+4;
+  /*--- Make sure to align the sizes with the constructor of CIncEulerVariable. ---*/
+  nVar = nDim+2; nPrimVar = nDim+9; nPrimVarGrad = nDim+6;
```

Comment:
"This little maneuver is going to cost us 51 years"
Similarly to what is done for Axisymmetric you could use the AuxVar to compute the extra gradient you need and that is not normally used.

Reply:
:) Yep changed that to use AuxVar instead and reverted nPrimVarGrad back to original